ofrak_core/Makefile

```diff
@@ -3,11 +3,11 @@ PIP=pip3
 
 .PHONY: install
 install: ofrak/gui/public
-	$(PIP) install .
+	$(PIP) install --pre .
```

Comment:
will this install pre-release versions for unpinned requirements?

Reply:
Good point -- now that I've commented out `ofrak-capstone` from test requirements this is not needed.